airbyte-webapp/src/components/ui/Table/Table.tsx

```diff
@@ -84,7 +84,7 @@ const Th = styled.th<TableHeaderProps>`
   font-size: ${({ light }) => (light ? 11 : 10)}px;
   line-height: 12px;
   color: ${({ theme, highlighted }) => (highlighted ? theme.whiteColor : theme.lightTextColor)};
-  border-bottom: ${({ theme, light }) => (light ? "none" : ` 1px solid ${theme.backgroundColor}`)};
+  border-bottom: none;
```

Comment:
:information_source: This border was anyway barely visible (and not sure why we at all tried to have a beige border below the dark blue table header), so I removed it.